schemas/context/personalidentifiers.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/personalidentifiers",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Personal identifiers",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "Condensed, normalized encapsulation of personal identifiers.",
+  "definitions": {
+    "personalidentifiers": {
+      "properties": {
+        "xdm:emailaddressid": {
```

Comment:
how about just `emailid`?

Reply:
this name is already used as an object name, thanks

Reply:
@rajindal  - Where is "emailid" used as an object name?

Reply:
@cmathis , it's used as object name in the end-user schema